### PR TITLE
Implement Loan Liquidation

### PR DESCRIPTION
**Pull request by SebastianBor**
Liquidations view #2

### CODE REVIEW

File: src/views/liquidate/item.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            {tokenName}
+          </span>
+          <div>
+            {formatNumber.format(borrowAmount)} {tokenName}
+          </div>
+          <div>
+            {formatPct.format(borrowAPY)}
+          </div>
+          <div>
+            {formatPct.format(ltv / 100)}
```
Comment (bartosz-lipinski) [inline src/views/liquidate/item.tsx:51]: I would just assume LTV is expressed as a number between 0 and 1

**Review by bartosz-lipinski — COMMENTED** [review]
Can you include screenshots and add to-do list of what else you would implement.

File: src/views/liquidate/item.tsx (hunk 1 above)
Comment (SebastianBor) [reply src/views/liquidate/item.tsx:51]: ok

**Comment by SebastianBor** [comment]
LiquidateView
LiquidateView
LiquidateReserveView
LiquidateReserveView

**Comment by SebastianBor** [comment]
Remaining tasks:
- Calculate LTV for liquidatable loans
- Wire liquidate() request to LiquidayeReserveView
- Support partial liquidation in LiquidayeReserveView

**Pull request merged**